priority-queue/PriorityQueueSender/QueueManager.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-namespace PriorityQueue.Shared
+namespace PriorityQueueSender
 {
     using System;
```

Comment:
Do we even need this file anymore?  Probably not if this moves over to output bindings.

Reply:
Removed indeed. Thanks.

Reply:
This file is gone, no needed anymore